src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +16,32 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(actor);
+            actor.setId((Long) session.getIdentifier(actor));
```

Comment:
Hibernate will automatically set a value for the "id" field. Hibernate has a rule that any entity stored in a session **must** have an id value
```suggestion
            actor.setId((Long) session.getIdentifier(actor));
```